### PR TITLE
Add SA quick start videos

### PR DESCRIPTION
**Pull request by EmilFattakhov**
- Linked on the SA installation page
- Linked on the updated intro page created by vexr

Please let me know if both placements make sense, thanks!

### CODE REVIEW

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 1]
```diff
@@ -33,54 +34,69 @@ What to expect:
...
+   - This is where Space Acres will store important data.
+   - Pick a location with plenty of free space; it might be as simple as **/Downloads**.
 
 3. **Choose the folder for farming and size of the farm**:
-    - This is where your **plots** (farming data) will be stored.
-    - The size determines your probability of getting a reward on the Network. In most cases, larger plots (or larger farm size) will gain you more rewards.
-    - If you prefer to have a few farms (e.g. you have some space space on different SSD drives), you can click on **Add farm** to pledge some additional storage space.
+   - This is where your **plots** (farming data) will be stored.
+   - The size determines your probability of getting a reward on the Network. In most cases, larger plots (or larger farm size) will gain you more rewards.
+   - If you prefer to have a few farms (e.g., you have some spare space on different SSD drives), you can click on **Add farm** to pledge some additional storage space.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:93]:
```suggestion
   - If you prefer to have a few farms (e.g. you have some spare space on different SSD drives), you can click on **Add farm** to pledge some additional storage space.
```

File: docs/farming-&-staking/farming/intro.mdx [hunk 2]
```diff
@@ -22,6 +22,11 @@ keywords:
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+:::tip Installation Videos
+
+For those who prefer a visual guide, we have prepared installation videos to assist you. The videos will help you to set up the Space Acres farming application in under 10 minutes. Please follow [the installation videos](/docs/farming-&-staking/farming/space-acres/space-acres-install.mdx#installation-videos) to install the wallet extension and refer to the detailed guide on downloading and configuring Autonomys' farming application, Space Acres.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/intro.mdx:27]:
```suggestion
For those who prefer a visual guide, we have prepared installation videos to assist you. The videos will help you to set up the Space Acres farming application in under 10 minutes. Please follow [the installation videos](/docs/farming-&-staking/farming/space-acres/space-acres-install.mdx#installation-videos) to use SubWallet to create a compatible Substrate account and refer to the detailed guide on downloading and configuring Autonomys' farming application, Space Acres.
```

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx [hunk 3]
```diff
@@ -33,54 +34,69 @@ What to expect:
...
-
 :::note
 Please note that a wallet with a Substrate address is required to run the software.<br />
 To learn how to create a Substrate wallet, please visit our pages on [SubWallet (recommended)](/docs/farming-&-staking/wallets/subwallet.md) or [**`{polkadot.js}`** extension](/docs/farming-&-staking/wallets/polkadot.md).
 :::
 
 
+### Installation Videos
+
+For those who prefer a visual guide, we have prepared installation videos to assist you. The videos will help you to set up Space Acres. Please select your operating system below to watch the corresponding installation video. If you prefer detailed text instructions, please proceed to [Step 1](#step-1-download-and-install-space-acres).
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:45]: Did you have the install video in the OS specific tabs originally? We only have one so far.

**Review by jim-counter — APPROVED** [review]
LGTM. Couple of suggestions.

File: docs/farming-&-staking/farming/space-acres/space-acres-install.mdx (hunk 3 above)
Comment (EmilFattakhov) [reply docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:45]: Sorry, yep they've been in tabs originally, but I though it was confusing. Adjusted the section 👍

Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/space-acres/space-acres-install.mdx:45]:
```suggestion
For those who prefer a visual guide, we have prepared installation videos to assist you. The videos will help you to set up Space Acres. If you prefer detailed text instructions, please proceed to [Step 1](#step-1-download-and-install-space-acres).
```

**Pull request merged**